src/main/java/edu/tamu/cap/model/response/Triple.java

```diff
@@ -59,19 +57,15 @@ public static Triple of(Map<String, String> tripleMap) {
     }
 
     public static Triple of(org.apache.jena.graph.Triple asTriple) {
-        return Triple.of(asTriple.getSubject().toString(), asTriple.getPredicate().toString(), asTriple.getObject().toString());
+        return Triple.of(asTriple.getSubject().toString(true), asTriple.getPredicate().toString(true), asTriple.getObject().toString(true));
     }
 
     @Override
     public String toString() {
         StringBuilder stngBldr = new StringBuilder();
         stngBldr.append("<").append(subject).append("> ");
         stngBldr.append("<").append(predicate).append("> ");
-        if (object.contains("http://") || object.contains("https://")) {
-            stngBldr.append("<").append(object).append("> ");
-        } else {
-            stngBldr.append("'").append(StringUtil.escape(StringUtil.removeQuotes(object))).append("' ");
-        }
+        stngBldr.append(object);
         return stngBldr.toString();
```

Comment:
This can be simplified to the following:

`return String.format("<%s> <%s> %s", subject, predicate, object)`